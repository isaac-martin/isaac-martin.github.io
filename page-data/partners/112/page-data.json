{"componentChunkName":"component---src-components-partner-landing-page-vc-template-jsx","path":"/partners/112/","result":{"pageContext":{"contentfulData":{"urlPath":"112","refCode":"11.2","pageDesign":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"7bJZnay0HFGMsosnHj8AW6","type":"Entry","createdAt":"2019-05-20T22:08:22.214Z","updatedAt":"2019-05-20T22:08:22.214Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"contentType":{"sys":{"type":"Link","linkType":"ContentType","id":"secondTierVc"}},"locale":"en-US"},"fields":{"designId":"Second Tier VC","vcName":"11.2 Capital","supportEmail":"11.2@brex.com","brexAffiliateLogo":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"1QFZrB33PzsD1xGPDQ7PBY","type":"Asset","createdAt":"2019-05-20T22:08:15.929Z","updatedAt":"2019-05-20T22:08:15.929Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"locale":"en-US"},"fields":{"title":"11.2","file":{"url":"//images.ctfassets.net/4wktpwxvwf0o/1QFZrB33PzsD1xGPDQ7PBY/003b96d30c550750e0195925e9707231/11.png","details":{"size":13165,"image":{"width":576,"height":136}},"fileName":"11.png","contentType":"image/png"}}}}}}}}}