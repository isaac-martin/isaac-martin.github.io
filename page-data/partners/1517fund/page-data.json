{"componentChunkName":"component---src-components-contentful-templates-partner-pages-partner-template-index-tsx","path":"/partners/1517fund/","result":{"pageContext":{"contentfulData":{"partnerType":"Startup","partnerName":"1517 Fund","urlPath":"1517fund","partnerLogo":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"5qk2pL5ylkszolr7KoUAnI","type":"Asset","createdAt":"2020-02-28T21:50:53.301Z","updatedAt":"2020-02-28T21:50:53.301Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"locale":"en-US"},"fields":{"title":"Screen Shot 2020-02-28 at 1.04.30 PM","file":{"url":"//images.ctfassets.net/4wktpwxvwf0o/5qk2pL5ylkszolr7KoUAnI/e3fb4577f3e7fd3ad4ea3834293f5a9f/Screen_Shot_2020-02-28_at_1.04.30_PM.png","details":{"size":38072,"image":{"width":846,"height":476}},"fileName":"Screen Shot 2020-02-28 at 1.04.30 PM.png","contentType":"image/png"}}},"useCombinedLogo":false,"refCode":"1517fund","headlineText":"The Corporate Card of Choice for","headlineEmphasis":"1517 Fund Companies","headlineSubtitle":"No personal guarantee, instant card access, and game-changing rewards. 1517 Fund companies get $500 in statement credit and waived card fees for life.","partnerQuote":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"7xBeH8kTxiY04GmwkH8hqW","type":"Entry","createdAt":"2020-02-07T18:26:28.625Z","updatedAt":"2020-02-07T18:26:28.625Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"contentType":{"sys":{"type":"Link","linkType":"ContentType","id":"partnerQuotes"}},"locale":"en-US"},"fields":{"quoteLogo":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"3M3Ox52YYhcxqK7XuiZC7z","type":"Asset","createdAt":"2020-02-07T18:25:51.016Z","updatedAt":"2020-02-07T18:25:51.016Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"locale":"en-US"},"fields":{"title":"Room Logo","file":{"url":"//images.ctfassets.net/4wktpwxvwf0o/3M3Ox52YYhcxqK7XuiZC7z/6f3ddca6b0e164a2caa56bef79882dca/room_logo_130x.webp","details":{"size":1830,"image":{"width":130,"height":130}},"fileName":"room_logo_130x.webp","contentType":"image/webp"}}},"quote":"Our finance team has both control over and visibility into employee spending. We aren’t a huge team right now, but we’re growing fast and I’m confident Brex’s program is built to scale.","quoteAuthor":"Brian Chen","authorTitle":"cofounder and CEO of ROOM"}},"finalHeading":"The only corporate card built for you."},"partnerType":"STARTUP"}}}