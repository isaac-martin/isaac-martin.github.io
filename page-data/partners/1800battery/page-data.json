{"componentChunkName":"component---src-components-partner-landing-page-ecommerce-big-partner-template-jsx","path":"/partners/1800battery/","result":{"pageContext":{"contentfulData":{"urlPath":"1800battery","refCode":"1800battery","pageDesign":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"4OkJHPEMlD8B3L7n5fx0rY","type":"Entry","createdAt":"2020-02-06T22:43:16.922Z","updatedAt":"2020-02-07T01:55:18.940Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":2,"contentType":{"sys":{"type":"Link","linkType":"ContentType","id":"ecommerceBigPartnerPage"}},"locale":"en-US"},"fields":{"designId":"Ecommerce Big Partner","brexAffiliateLogo":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"4EHfrZVPVFRxt1fh2hfyNH","type":"Asset","createdAt":"2020-02-06T22:41:57.977Z","updatedAt":"2020-02-06T22:41:57.977Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"locale":"en-US"},"fields":{"title":"Group 538","file":{"url":"//images.ctfassets.net/4wktpwxvwf0o/4EHfrZVPVFRxt1fh2hfyNH/c19f53f5f0b8612bbba17178648b1105/Group_538.png","details":{"size":42221,"image":{"width":798,"height":122}},"fileName":"Group 538.png","contentType":"image/png"}}},"subtitle":"1800 Battery referrals receive a $1000 signup bonus and waived card fees for life","partnerName":"1800 Battery"}}}}}}