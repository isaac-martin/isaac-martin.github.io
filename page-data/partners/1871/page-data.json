{"componentChunkName":"component---src-components-partner-landing-page-vc-template-jsx","path":"/partners/1871/","result":{"pageContext":{"contentfulData":{"urlPath":"1871","refCode":"1871","pageDesign":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"5K2qw6CRoUY0o82IsNw4a9","type":"Entry","createdAt":"2019-07-15T21:28:36.751Z","updatedAt":"2019-07-15T21:28:36.751Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"contentType":{"sys":{"type":"Link","linkType":"ContentType","id":"secondTierVc"}},"locale":"en-US"},"fields":{"designId":"Second Tier VC","vcName":"1871","supportEmail":"1871@brex.com","brexAffiliateLogo":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"a5VBp1ZojA3qCkm75yYDy","type":"Asset","createdAt":"2019-07-15T21:28:22.712Z","updatedAt":"2019-07-15T21:28:22.712Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"locale":"en-US"},"fields":{"title":"1871","file":{"url":"//images.ctfassets.net/4wktpwxvwf0o/a5VBp1ZojA3qCkm75yYDy/58baa530b6d15bb9a9bbfa5fa9a3b906/Group_212.png","details":{"size":9290,"image":{"width":630,"height":136}},"fileName":"Group 212.png","contentType":"image/png"}}}}}}}}}