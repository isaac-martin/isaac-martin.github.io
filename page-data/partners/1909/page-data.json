{"componentChunkName":"component---src-components-partner-landing-page-vc-template-jsx","path":"/partners/1909/","result":{"pageContext":{"contentfulData":{"urlPath":"1909","refCode":"1909","pageDesign":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"7f6rkWnAZEG94dKJ4C1neW","type":"Entry","createdAt":"2019-11-25T16:26:20.393Z","updatedAt":"2019-11-25T16:26:20.393Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"contentType":{"sys":{"type":"Link","linkType":"ContentType","id":"firstTierVc"}},"locale":"en-US"},"fields":{"brexAffiliateLogo":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"3nxkP4QXYtXb2ufXrY4XJB","type":"Asset","createdAt":"2019-11-25T16:26:15.216Z","updatedAt":"2019-11-25T16:26:15.216Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"locale":"en-US"},"fields":{"title":"1909","file":{"url":"//images.ctfassets.net/4wktpwxvwf0o/3nxkP4QXYtXb2ufXrY4XJB/3f6a08a33389f2d8c423b26ef91a36ca/Group_14.png","details":{"size":12501,"image":{"width":855,"height":158}},"fileName":"Group 14.png","contentType":"image/png"}}},"designId":"First Tier VC","statementCredit":500,"supportEmail":"1909@brex.com","vcName":"1909"}}}}}}