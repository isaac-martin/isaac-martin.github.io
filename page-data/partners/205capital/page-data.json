{"componentChunkName":"component---src-components-partner-landing-page-vc-template-jsx","path":"/partners/205capital/","result":{"pageContext":{"contentfulData":{"urlPath":"205capital","refCode":"205capital","pageDesign":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"6HhNV1Yvg156Qcm2PSf1NH","type":"Entry","createdAt":"2019-11-15T23:04:57.321Z","updatedAt":"2019-11-15T23:04:57.321Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"contentType":{"sys":{"type":"Link","linkType":"ContentType","id":"secondTierVc"}},"locale":"en-US"},"fields":{"designId":"Second Tier VC","vcName":"205 Capital","supportEmail":"205capital@brex.com","brexAffiliateLogo":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"78pSNm6rRq17XoVmEyAfGy","type":"Asset","createdAt":"2019-11-15T23:04:53.474Z","updatedAt":"2019-11-15T23:04:53.474Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"locale":"en-US"},"fields":{"title":"205","file":{"url":"//images.ctfassets.net/4wktpwxvwf0o/78pSNm6rRq17XoVmEyAfGy/b3889c7a3bb66258946e89e28a4d58bd/Group_457.png","details":{"size":25076,"image":{"width":587,"height":136}},"fileName":"Group 457.png","contentType":"image/png"}}}}}}}}}