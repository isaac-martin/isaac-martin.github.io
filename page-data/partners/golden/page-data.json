{"componentChunkName":"component---src-components-partner-landing-page-vc-template-jsx","path":"/partners/golden/","result":{"pageContext":{"contentfulData":{"urlPath":"golden","refCode":"golden","pageDesign":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"4bkmcyh4mjAAoMuJnWAZDq","type":"Entry","createdAt":"2019-05-20T22:35:45.331Z","updatedAt":"2019-05-20T22:35:45.331Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"contentType":{"sys":{"type":"Link","linkType":"ContentType","id":"secondTierVc"}},"locale":"en-US"},"fields":{"designId":"Second Tier VC","vcName":"Golden Ventures","supportEmail":"golden@brex.com","brexAffiliateLogo":{"sys":{"space":{"sys":{"type":"Link","linkType":"Space","id":"4wktpwxvwf0o"}},"id":"5iQ7wFZCx329RAu4L8zV7V","type":"Asset","createdAt":"2019-05-20T22:35:39.586Z","updatedAt":"2019-05-20T22:35:39.586Z","environment":{"sys":{"id":"master","type":"Link","linkType":"Environment"}},"revision":1,"locale":"en-US"},"fields":{"title":"Golden","file":{"url":"//images.ctfassets.net/4wktpwxvwf0o/5iQ7wFZCx329RAu4L8zV7V/cca56242016514c67a3c304179b8b349/golden.png","details":{"size":14644,"image":{"width":799,"height":136}},"fileName":"golden.png","contentType":"image/png"}}}}}}}}}